desktop/src/gui/picker.rs

```diff
@@ -34,10 +38,17 @@ impl FilePicker {
             return None;
         }
 
+        let locale = &self.data.preferences.language();
         let mut dialog = AsyncFileDialog::new()
-            .add_filter("Flash Files", &["swf", "spl", "ruf"])
-            .add_filter("All Files", &["*"])
-            .set_title("Load a Flash File");
+            .add_filter(
+                text(locale, "file-picker-filter-supported"),
+                &["swf", "spl", "ruf"],
+            )
+            .add_filter(text(locale, "file-picker-filter-swf"), &["swf"])
+            .add_filter(text(locale, "file-picker-filter-spl"), &["spl"])
+            .add_filter(text(locale, "file-picker-filter-ruf"), &["ruf"])
+            .add_filter(text(locale, "file-picker-filter-all"), &["*"])
+            .set_title(text(locale, "picker-title-open-file"));
```

Comment:
Shouldn't this be `file-picker-title-open-file`?

Reply:
I have no idea how I missed that :scream: